src/app/search/search.component.ts

```diff
@@ -27,6 +27,7 @@ import { ELASTIC_SEARCH_CLIENT } from './elastic-search-client';
 import { QueryBuilderService } from './query-builder.service';
 import { SearchQuery } from './state/search.query';
 import { SearchService } from './state/search.service';
+import { Hit } from './state/search.service';
```

Comment:
This should be part of the previous line: `import { Hit, SearchService } from './state/search.service`.

If you let WebStorm import instead of manually adding, this will typically just work.

Reply:
Done!